content/blog/2020-07-16-devops-for-data-scientists.md

```diff
@@ -127,7 +127,7 @@ Git refresher?
 When a feature branch passes the automated tests, it becomes a candidate to be
 merged into the master branch.
 
-![](/static/uploads/2020-07-16/basic_ci_system.png) _Here's what continuous
+![](/uploads/images/2020-07-16/basic_ci_system.png) _Here's what continuous
```

Comment:
hmm ... why is it working w/o this change?

Reply:
This one confused me as well, I think something in the Gatsby pipeline that I'm not familiar with automatically corrected this.
The `/static/uploads/` version of the path doesn't map to a real directory on the filesystem nor on the site, but the images do get picked up by `gatsby-image` without the change.